### PR TITLE
Major Updates

### PR DESCRIPTION
**Pull request by sku146**
### Major Updates
- Updated Webpack version to 4.x.x
- changed extract-text plugin to min css extract plugin
- added Webpack command module
- Changed the engine configs

### CODE REVIEW

**Comment by sku146** [comment]
rebase branch

File: package.json [hunk 1]
```diff
@@ -36,53 +38,61 @@
   "license": "ISC",
   "engines": {
     "node": ">= 6",
```
Comment (otorrillas) [inline package.json:40]: Shouldn't this be node 7? Node 6 doesn't come with npm >= 4 built-in.

File: package.json [hunk 2]
```diff
@@ -36,53 +38,61 @@
...
-    "http-server": "^0.10.0",
+    "csp-html-webpack-plugin": "^2.3.0",
+    "css-loader": "^0.28.11",
+    "eslint-config-sxt-engine": "^1.1.0",
+    "favicons-webpack-plugin": "0.0.9",
+    "file-loader": "^1.1.11",
+    "html-webpack-plugin": "^3.2.0",
+    "http-server": "^0.11.1",
     "ink-docstrap": "^1.3.2",
-    "jest": "^21.2.1",
```
Comment (otorrillas) [inline package.json:61]: 👍 cool!

File: package.json [hunk 3]
```diff
@@ -36,53 +38,61 @@
...
   "engines": {
     "node": ">= 6",
-    "npm": ">= 3"
+    "npm": ">= 4"
   },
   "dependencies": {
-    "babel-jest": "^21.2.0",
-    "babel-loader": "^7.1.2",
-    "babel-preset-build-engine": "^1.0.1",
+    "babel-jest": "^23.0.1",
```
Comment (otorrillas) [inline package.json:44]: This should match `jest` version, though...

File: package.json (hunk 1 above)
Comment (otorrillas) [reply package.json:40]: This or, simply:

```
"node": ">=8"
"npm": ">=5"
```

Which is much faster, comes with built-in package-lock.json, etc.

**Review by otorrillas — COMMENTED** [review]
Some review comments regarding packages installed.

**Pull request closed without merging**